components/Calendar.js

```diff
@@ -68,6 +75,12 @@ export default class Calendar extends Component {
     weekStart: 1,
   };
 
+  constructor(props) {
+    super(props);
+
+    this.onWeekRowLayout = this.onWeekRowLayout.bind(this);
+  }
+
```

Comment:
You can remove this constructor and bind if you update line 172 to
`onWeekRowLayout = (event) => {}`